.github/ISSUE_TEMPLATE.md

```diff
@@ -26,7 +26,9 @@ Environment data
     } else {
         "os: $((dir $env:SystemRoot\System32\cmd.exe).VersionInfo.FileVersion)"
     }
-    "PS file version: $((dir $pshome\p*[hl].exe).VersionInfo.FileVersion)"
+    "PS file version: $((dir $pshome\p*[hl][.exe]*).VersionInfo.FileVersion)"
```

Comment:
> dir $pshome\p*[hl][.exe]*

This results in a list of files instead of the executable file only.

```powershell
PS:24> dir $pshome\p*[hl][.exe]*


    Directory: C:\Program Files\PowerShell\6

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----         1/18/2019  9:46 PM             57 powershell.config.json
-a----         3/22/2019  1:12 AM          92160 PowerShell.Core.Instrumentation.dll
-a----         2/13/2019  5:15 PM         220724 PowerShell.Core.Instrumentation.man
-a----         3/25/2019  5:38 PM         170494 pwsh.deps.json
-a----         3/25/2019  5:42 PM          14712 pwsh.dll
-a----         3/25/2019  5:42 PM         276344 pwsh.exe
-a----         3/25/2019  5:38 PM          15872 pwsh.pdb
-a----         3/25/2019  5:38 PM            152 pwsh.runtimeconfig.json
-a----         3/25/2019  5:38 PM            627 pwsh.xml
```

Reply:
```suggestion
    "PS file version: $($name = if ($PSVersionTable.PSEdition -eq "Core") { "pwsh.dll" } else { "powershell.exe" }; (dir $pshome\$name).VersionInfo.FileVersion)"
```